compiler/lib/generate.ml

```diff
@@ -1062,6 +1062,7 @@ let rec translate_expr ctx queue loc x e level : _ * J.statement_list =
       (Mlvalue.Block.field cx n, or_p px mutable_p, queue), []
   | Closure (args, ((pc, _) as cont)) ->
       let loc = source_location ctx ~force:After (After pc) in
+      let fv = Addr.Map.find pc ctx.freevars in
```

Comment:
Does `ctx.freevars` list all free variables, or just the ones bound inside a loop?